reactor-netty-http/src/main/java/reactor/netty/http/client/HttpConnectionProvider.java

```diff
@@ -56,6 +56,15 @@ else if (http1ConnectionProvider == null) {
 		}
 	}
 
+	@Override
+	public Builder mutate() {
+		if (this.http1ConnectionProvider == null) {
+			return ConnectionProvider.super.mutate();
+		}
+
+		return this.http1ConnectionProvider.mutate();
```

Comment:
I know that I haven't considered `h2ConnectionProvider` yet.
I'd like to hear the opinion of the maintainer.